e2e/test.js

```diff
@@ -12,9 +12,14 @@ test("landing", async ({ page, baseURL }, testInfo) => {
   await expect(page).toHaveScreenshot("landing-top.png")
 
   if (testInfo.project.name != "ios") {
+    // let's make sure we don't take snapshots of the gif
```

Comment:
I don't understand this comment (what gif)?. But perhaps it is better to say what you *are* testing rather than what you are not.

Reply:
ok, i didn't read very clearly. I see this test is testing multiple images on the page and that this part is the playground displayed on the landing page.